repository/src/main/java/org/apache/atlas/tasks/TaskManagement.java

```diff
@@ -130,8 +130,8 @@ public void addFactory(TaskFactory taskFactory) {
         createTaskTypeFactoryMap(this.taskTypeFactoryMap, taskFactory);
     }
 
-    public AtlasTask createTask(String taskType, String createdBy, Map<String, Object> parameters) {
-        return this.registry.createVertex(taskType, createdBy, parameters);
+    public AtlasTask createTask(String taskType, String createdBy, Map<String, Object> parameters, String classificationId, String entityGuid) {
```

Comment:
can't we create a new method with more params and same name?

Reply:
Addresses this one